src/interfaces/ast/profile-ast.ts

```diff
@@ -115,6 +115,7 @@ export type Type = TypeName | TypeDefinition;
  */
 export interface EnumValueNode extends ProfileASTNodeBase, DocumentedNode {
   kind: 'EnumValue';
+  name: string;
```

Comment:
is it necessary to make it required? i'd rather not break everything again

Reply:
I don't like the idea of making our type definitions bad for legacy reasons. The `name` is always going to be present, no matter what.

We can wait with this PR until more things come along that break something 🤷‍♀️

Reply:
The name is not always going to be present, it will not be present in an ast parsed with older version of the parser. I don't like the idea of breaking everything for everyone for no reason. The name is just a cosmetic stuff, it's not really needed by users and therefore it's not 'bad' type definition if it's optional (even though it will be present in all future releases)

Reply:
It's always going to be present from a certain version is what I meant.

It is needed by the GraphQL generator so it's not just for cosmetic stuff.

But you are right, the most sensible move is to make it optional since we have complex interactions with the ecosystem.

Reply:
I meant it's not needed by users